os/drivers/wireless/realtek/os/tizenrt/tizenrt_service.c

```diff
@@ -592,7 +592,16 @@ static int _tizenrt_create_task(struct task_struct *ptask, const char *name,
 								u32 stack_size, u32 priority, thread_func_t func, void *thctx)
 {
 	ptask->task_name = name;
-	ptask->task = kernel_thread(name, priority, stack_size, (main_t)func, NULL);
+	u32 priority_tmp = priority;
+	if (!strncmp(name, "recv_thread", sizeof("recv_thread"))) {
+		priority_tmp = 224;
```

Comment:
Instead of constant number, CONFIG_SCHED_HPWORKPRIORITY or CONFIG_SCHED_LPWORKPRIORITY will be better to align with another thread.